packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -115,7 +159,16 @@ export class SObjectDescribe {
     }
   }
 
-  public getVersion(): string {
-    return `${this.versionPrefix}${this.targetVersion}`;
+  public async fetchObjects(types: string[]): Promise<SObject[]> {
```

Comment:
Moving this function here allows us to do all sobject describe requests from this class.